spherogram_src/links/tangles.py

```diff
@@ -248,6 +248,7 @@ def __init__(self, a, b=1):
 
 class IdentityBraid(Tangle):
     def __init__(self, n):
+        assert n >= 0
```

Comment:
Like @culler suggested, let's have this give us a better error message. For example,
```suggestion
        if n < 0: raise ValueError("The number of strands must be non-negative.")
```